### PR TITLE
Db/seed data

### PR DESCRIPTION
**Pull request by TripolarHZ**
finished creation of all tables

### CODE REVIEW

File: db/seedData.js [hunk 1]
```diff
@@ -85,6 +87,29 @@ async function createTables() {
         user_id INTEGER REFERENCES users(id),
         billing_address_id INTEGER REFERENCES billing_addresses(id)
```
Comment (cmceachin5888) [inline db/seedData.js:88]: change to "shipping"

File: db/models/users.js [hunk 2]
```diff
@@ -6,12 +6,8 @@ const { addShippingAddressToUser, getShippingAddressByUserId } = require('./ship
 // database functions
 
 // user functions
-async function createUser({
-  name,
-  email,
-  password,
-  addresses: { billingAddressList, shippingAddressList },
-}) {
```
Comment (cmceachin5888) [inline db/models/users.js:14]: i had to make several updates to Users.js as well. maybe you can revert yours so we dont conflict?

your change here for the addresses, whats the thought here? i have the tests passing as is, so i would need to edit them to accommodate.

**Review by cmceachin5888 — CHANGES_REQUESTED** [review]
let me know what you think

**Review by Scarlette-O — APPROVED** [review]

**Review by Vimaser — APPROVED** [review]
Looks good. Checked in zoom.

**Review by cmceachin5888 — APPROVED** [review]

**Pull request merged**